lib/ts/recipe/session/recipe.ts

```diff
@@ -118,6 +128,22 @@ export default class SessionRecipe extends RecipeModule {
         SessionRecipe.instance = undefined;
     }
 
+    static addClaimFromOtherRecipe = (builder: SessionClaim<any>) => {
+        SessionRecipe.claimsAddedByOtherRecipes.push(builder);
+    };
+
+    static getClaimsAddedByOtherRecipes = (): SessionClaim<any>[] => {
+        return SessionRecipe.claimsAddedByOtherRecipes;
+    };
+
+    static addClaimValidatorFromOtherRecipe = (builder: SessionClaimValidator) => {
+        SessionRecipe.claimValidatorsAddedByOtherRecipes.push(builder);
```

Comment:
I don't think we need to since this is only used internally, it should be easy to keep track of. Basically these should only be added a single time at the end of the supertokens init.

Reply:
Well.. in that case, if we are passing in the same validator twice, we should throw an error - might help find bugs. I wouldn't want to just ignore this type of issue cause it's only meant for internal use.

Reply:
So there might be a case where we are adding the same validators with different values, for example, BooleanClaims.validators.isTrue() and BooleanClaims.validators.isFalse() -> whilst they are checking for different conditions, they use the same validators. Therefore, we decided not to throw and keep it the way it is.